odd_collector/adapters/cassandra/repository.py

```diff
@@ -0,0 +1,109 @@
+import contextlib
+import logging
+from abc import ABC, abstractmethod
+from typing import List
+
+import cassandra
+from cassandra.auth import PlainTextAuthProvider
+from cassandra.cluster import EXEC_PROFILE_DEFAULT, Cluster, ExecutionProfile
+from cassandra.query import tuple_factory
+
+
+TABLE_METADATA_QUERY: str = """
+    SELECT * FROM system_schema.tables 
+    WHERE keyspace_name = %(keyspace)s;
+"""
+
+COLUMNS_METADATA_QUERY: str = """
+    SELECT * FROM system_schema.columns 
+    WHERE keyspace_name = %(keyspace)s;
+"""
+
+VIEWS_METADATA_QUERY: str = """
+    SELECT * FROM system_schema.views 
+    WHERE keyspace_name = %(keyspace)s;
+"""
+
+
+class AbstractRepository(ABC):
```

Comment:
Write types what method return.
AbstractRepository class should return types from [odd_collector/adapters/cassandra/mappers/models.py](https://github.com/opendatadiscovery/odd-collector/pull/139/files#diff-4ebc14bf4d8743b8298aec29889caa1af0ee54cea02fbb2c0e1b5e1bd77bc344)
i.e
```python
class AbstractRepository(ABC):
    @abstractmethod
    def get_tables(self) -> List[TableMetadata]:
        pass
```